src/network_analysis.jl

```diff
@@ -399,7 +398,7 @@ end
 # Used in the subsequent function.
 function subnetworkmapping(linkageclass, allrxs, complextorxsmap, p)
     rxinds = sort!(collect(Set(rxidx for rcidx in linkageclass
-                               for rxidx in complextorxsmap[rcidx])))
+    for rxidx in complextorxsmap[rcidx])))
```

Comment:
```suggestion
        for rxidx in complextorxsmap[rcidx])))
```
This reformat needs manual updating as the code is now misleading with the loss of indentation.